lib/ui/common/widget/primary_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+const _radius = 8.0;
+const _elevation = 0.0;
+const _backgroundColor = Color(0xFFE0E0E0);
+const _paddingHorizontal = 10.0;
+const _paddingVertical = 5.0;
+
+class PrimaryButton extends StatelessWidget {
+  final String text;
+  final Function onPressed;
+
+  const PrimaryButton({
+    @required this.text,
+    @required this.onPressed,
+  }) : assert(onPressed != null);
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
fat arrow? używamy w buildzie? kwestia ustalenia

Reply:
Proponowałbym, żeby fat arrow używać kiedy mamy do czynienia z czymś krótkim i zwięzłym - w przypadku co najmniej kilku zagnieżdżonych linijek formatowanie z fat arrow imo wygląda nawet mniej czytelnie. Także generalnie jak zwykle faktycznie build będzie nieco dłuższy więc proponowałbym jednak w takich przypadkach {}, ale z drugiej strony jak build będzie się sprowadzał do => Text("...") to grzech nie użyć fat arrow :D

Reply:
ok zgadzam się, fat arrow jak to jest krótkie i zwięzłe